lib/interceptor/cache.js

```diff
@@ -166,11 +248,24 @@ module.exports = (opts = {}) => {
             )
           )
         }
+
+        // Dump request body.
+        if (util.isStream(opts.body)) {
+          opts.body.on('error', () => {}).destroy()
+        }
```

Comment:
Operations that mutates sates, usually are non-safe methods and they cannot be cached isn't it? If body received, we are mutating the state so possibly we wouldn't like to cache this request.

am I getting it wrong?